### ISSUE TITLE
Maybe delete /tlg4034/tlg006/tlg4034.tlg006.opp-grc2.xml

### ISSUE THREAD

**Issue by sonofmun** [OpenGreekAndLatin/First1KGreek#155]

**Comment by annettegessner** [comment]
this text and the text from #154 are exactly the same and even linking to the same scan online,
the other one (opp-grc1) only has a slightly better structure

**Comment by annettegessner** [comment]
I suggest deleting this text here!?

**Comment by brittanyhardy** [comment]
Tried deleting duplicate document (and traces of it in the CTS metadata file, etc.), but the pushed files failed the Travis test with this error message:

|          Identifier         | Words  | Nodes |      Failed Tests      |
+-----------------------------+--------+-------+------------------------+
| tlg4034.tlg006.opp-grc2.xml | 28,727 |   1   | Available in inventory

**Comment by sonofmun** [comment]
@brittanyhardy This suggests that you deleted the part of the `__cts__.xml` metadata file that refers to tlg4034.tlg006.opp-grc2.xml . Check to make sure that the URN that is on the `<div type="edition">` element in tlg4034.tlg006.opp-grc2.xml is mentioned in the `__cts__.xml` file.

**Comment by brittanyhardy** [comment]
What should we do in the case that we deleted the above file (opp-grc2)? This issue calls for that file, which is a duplicate, to be deleted. Thanks!

**Comment by sonofmun** [comment]
How is the file being tested in HookTest (you show the results above) if it doesn't exist?

**Comment by brittanyhardy** [comment]
We have no idea. We deleted the file and then even removed it from trash, so it is completely gone from my computer, yet it still appears in the HookTest. Any thoughts on why?

**Comment by sonofmun** [comment]
Point me to the repository that is being tested, e.g., your fork of 1st1K, and the branch (could be your master branch).

**Comment by brittanyhardy** [comment]
I've attached a screenshot of my issue branch in Terminal where you can see that grc2 has been deleted. The issue branch is here https://github.com/brittanyhardy/First1KGreek/branches issue-155
![screen shot 2017-06-08 at 3 26 49 pm](https://user-images.githubusercontent.com/29205107/26947017-2c85aa64-4c5f-11e7-9141-8a40c0c6897b.png)

**Comment by sonofmun** [comment]
On your issue-155 branch on your repository, the opp-grc2 file still exists. See https://github.com/brittanyhardy/First1KGreek/tree/issue-155/data/tlg4034/tlg006
But it is not mentioned in the `__cts__.xml` file. So you need to make sure to push your local changes to your issue-155 branch, which should delete the opp-grc2 file. Just make sure that you commit the changes locally to your issue-155 branch, not your master branch. On your remote repository, the opp-grc2 file does not exist on your master branch. My guess is that you made the local changes to your master branch and then pushed them to your remote master branch but not to your issue-155 branch.
I think the easiest solution at this point would be to switch locally to your issue-155 branch, delete the file there, then push to your remote issue-155 branch, make sure that it passes HookTest on Travis, and then do a pull request from that branch to OpenGreekAndLatin:master

**Comment by brittanyhardy** [comment]
The file pop-grc2 is not in my master branch nor my issue branch (I've attached screenshots checking out both branches which show that the file was deleted). We have no idea why it is showing up in that repository online.
![screen shot 2017-06-12 at 2 38 13 pm](https://user-images.githubusercontent.com/29205107/27049314-f430c36c-4f7c-11e7-8b96-af64e580c251.png)
![screen shot 2017-06-12 at 2 37 00 pm](https://user-images.githubusercontent.com/29205107/27049315-f43f9a54-4f7c-11e7-947d-01a3124aa8a1.png)

**Comment by sonofmun** [comment]
You need to update your local issue-155 branch with your remote one. Try these commands from your First1K directory:

```
git checkout issue-155
git add -A
git commit -m "Deleted unnecessary file"
git push issue-155
```
In that last command, you may need to specify the name of your remote. You can find this out with the command:
```
git remote -v
```
Which will show output something like this:
```
origin  https://github.com/OpenGreekAndLatin/First1KGreek.git (fetch)
origin  https://github.com/OpenGreekAndLatin/First1KGreek.git (push)
```
In this case, the `push` repo name is `origin`. So then the final command in the first sequence would be:
```
git push origin issue-155
```
I hope that helps.

**Comment by annettegessner** [comment]
Please note here if the deletion was successful and mention the PR it's been done in like this:
see #1536

**Comment by sonofmun** [comment]
Corrected with #1617
